tests/explorer.api.tests/ExploreTest.cs

```diff
@@ -13,7 +14,7 @@ public sealed class ExploreTest
     {
         private const string apiKey = "SFMyNTY.g3QAAAACZAAEZGF0YW0AAAAkMTUxNjJiZWYtNWE2MS00NGNhLWFiZmUtOWU1MGFiNGIxM2M4ZAAGc2lnbmVkbgYAlH8NpW8B.byOGmraal0gWNKa_g6aXgArfff2nl34Tm-hJL43sOIw";
```

Comment:
Is it safe to store the API key here?

Reply:
Yeah totally safe, it'll never get lost. 😆 

I suppose best practice here is to pass it in as an environment variable or have a local config file that is never checked in to source control. However then I can't see how we would get the api key into Travis CI for testing..

Reply:
I also don't know how to get API keys into Travis CI without involving git. It seems that there is some documentation here https://docs.travis-ci.com/user/private-dependencies/ but I am not sure if it is worth investigating it. @sebastian  what do you think about this?

Reply:
Definitely not safe to store API keys here.
This key has already been compromised and should be immediately revoked.
You can set environmental variables for Travis to use on Travis itself.

Reply:
We do not need the private dependencies features. It's for testing private repositories whereas this is a public one.

Please check if you have access to the Travis Explorer repo config when logging in to travis using your github credentials. If you do, then you can set the environmental variable there. If for some reason it does not work, let me know, and pass me a key via an out of bounds mechanism that I can add for you instead.

Reply:
This is a URL you can use to check if you have access. It's also where the setting would be changed: https://travis-ci.org/diffix/explorer/settings

Reply:
Thanks for the link, didn't know we had that! I've generated a new key and added it to the environment, will test it now.

Reply:
Ok all set up now. 

@AndreiBozantan if you want to run the tests locally you need an API key added to your environment (https://attack.aircloak.com/api_tokens) under AIRCLOAK_API_KEY. You probably also have to rebuild the development container. 

Fyi. The link should be: https://travis-ci.org/diffix/explorer/settings
(note .com instead of .org, apparently .org is being phased out).